lib/twilio/uuid.ts

```diff
@@ -28,7 +28,7 @@ function generateUuid(): string {
 
   const generateRandomValues: () => string =
     typeof crypto.randomUUID === 'function'
-      ? crypto.randomUUID
+      ? () => crypto.randomUUID!()
```

Comment:
I see, this is where the illegal invocation was occurring? You could also do ```
? crypto.randomUUID?.bind(crypto)
``` but it doesn't make too much of a difference.